include/Zycore/Types.h

```diff
@@ -333,6 +333,84 @@ typedef char* ZyanCharPointer;
  */
 typedef const char* ZyanConstCharPointer;
 
+/* ============================================================================================== */
+/* Type conversion utils                                                                          */
```

Comment:
Perhaps this is not a perfect place but it's the best one from existing ones. Fallback functions require types to be properly defined so `Defines.h` is not an option.

If we don't want it here I can move this and other utilities from `Defines.h` to a new header file (`Utils.h`? `Arithmetic.h`?).

Reply:
I think this is fine as-is. :)